pilot/pkg/model/context.go

```diff
@@ -308,12 +309,33 @@ func (s *StringBool) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+// ProxyConfig can only be marshalled using (gogo) jsonpb. However, the rest of node meta is not a proto
```

Comment:
Why is this in model ? Can it move to one of the many helper packages ?

Reply:
model is imported basically everywhere, its best to avoid dependencies in model, since all the helper packages import it so there are cyclic imports. Any concern with keeping this here? It must be in the same package as the type is defined due to how go works I think